ember-client/shared/src/main/scala/org/http4s/ember/client/internal/ClientHelpers.scala

```diff
@@ -286,7 +286,7 @@ private[client] object ClientHelpers {
         case Left(_: ClosedChannelException) => true
         case Left(ex: IOException) =>
           val msg = ex.getMessage()
-          msg == "Connection reset by peer" || msg == "Broken pipe"
+          msg == "Connection reset by peer" || msg == "Broken pipe" || msg == "Connection reset"
```

Comment:
Maybe should just check if `msg` contains `Connection reset` ?

Reply:
Hello. You are probably. I just propose the less impactful solution that is working for us.

Reply:
Drive-by comment, but here's some of [Netty's logic](https://github.com/netty/netty/blob/217df76c40be0f9d4f82e8cf05d68ea07a23f42f/handler/src/main/java/io/netty/handler/ssl/SslHandler.java#L1215), which includes a hefty regex that may be inspirational.

Reply:
Hello. 
What is blocking to merge this fix?
It could be cool to avoid duplicate this fix in all our projects.
And i think it could be useful for other production applications.
Best regards